components/users/ui/src/main/resources/PhenoTips/PhenoTipsGroupSheet.xml

```diff
@@ -209,7 +214,7 @@ $xwiki.ssfx.use('icons/font-awesome/css/font-awesome.min.css')##
       <cache>long</cache>
     </property>
     <property>
-      <code>$("manage-administrators") &amp;&amp; $("manage-administrators").observe("click", function(e) {
+      <code>$("manage-administrators") &amp;&amp; $("manage-managers").observe("click", function(e) {
```

Comment:
You still use `$("manage-administrators")` on the left, which will break the condition.